internal/queries/query_builder.go

```diff
@@ -99,11 +99,6 @@ func (qb *QueryBuilder) QuerySingle(entityId string) (*api.Entity, error) {
 		return nil, nil
```

Comment:
This should probably be
```
if len(entity) < 2 {
    return nil, errors.New("unexpected response")
}
```

reasoning:
- valid responses always consist of 2 entities: 1. context + 2. the query result
- returning nil,nil indicates that there is nothing wrong. which is not the case here